src/control/server/ctl_system.go

```diff
@@ -32,6 +34,8 @@ import (
 	"github.com/daos-stack/daos/src/control/system"
 )
 
+var systemRPCMaxDuration = 30 * time.Second
```

Comment:
Could maybe call this systemRequestTimeout for consistency with the above naming suggestion? I also wonder if this will need to be set dynamically based on system size. I guess we can adjust as we get into real scale testing.